### PR TITLE
feat(react): add ActionCard component

### PR DESCRIPTION
**Pull request by thivi**
### Purpose

> Create `ActionCard` component
image

### Related Issues
- Issue https://github.com/wso2/oxygen-ui/issues/2

### Related PRs
- Related PR `#1` or (None)

### Checklist
- [ ] UX/UI review done on the final implementation.
- [ ] Story provided. (Add screenshots)
- [ ] Manual test round performed and verified.
- [ ] Unit tests provided. (Add links if there are any)
- [ ] Documentation provided. (Add links if there are any)

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: packages/react/src/components/ActionCard/ActionCard.tsx [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+import Card from '@mui/material/Card';
+import CardActions from '@mui/material/CardActions';
+import CardContent from '@mui/material/CardContent';
+import {FC, ReactElement, ReactNode} from 'react';
+import Button from '../Button';
+import Typography from '../Typography';
+import {WithWrapperProps} from '../../models';
+
+export interface ActionCardProps {
+  actionText: string;
```
Comment (brionmario) [inline packages/react/src/components/ActionCard/ActionCard.tsx:28]: Shall we document the props?

File: packages/react/src/components/ActionCard/ActionCard.tsx [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+        <Typography
+          variant="body1"
+          color="secondary.main"
+          sx={{
+            color: '#8D91A3',
+            fontFamily: 'Montserrat',
+            fontSize: '13px',
+            fontStyle: 'normal',
+            fontWeight: '400',
+            lineHight: '20px',
```
Comment (thivi) [inline packages/react/src/components/ActionCard/ActionCard.tsx:86]: @brionmario Can you review this?

File: packages/react/src/components/ActionCard/ActionCard.tsx [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+  /**
+   * The title of the card.
+   */
+  title: string;
+}
+
+const COMPONENT_NAME: string = 'ActionCard';
+
+const ActionCard: FC<ActionCardProps> & WithWrapperProps = (props: ActionCardProps): ReactElement => {
+  const {image, title, description, actionText, onActionClick} = props;
```
Comment (brionmario) [inline packages/react/src/components/ActionCard/ActionCard.tsx:53]: Shall we spread `rest` in the root `Card` component.
```suggestion
  const {image, title, description, actionText, onActionClick, ...rest} = props;
```

File: packages/react/src/components/ActionCard/ActionCard.tsx [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+
+  return (
+    <Card
+      sx={{
```
Comment (brionmario) [inline packages/react/src/components/ActionCard/ActionCard.tsx:57]: Shall we move this to `action-card.scss`?

File: packages/react/src/components/ActionCard/__tests__/ActionCard.test.tsx [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+ * under the License.
+ */
+
+import {render} from '@unit-testing';
+import {ReactElement} from 'react';
+import ActionCard from '../ActionCard';
+
+const onActionClick: jest.Mock<any, any> = jest.fn();
+
+const actionCard: ReactElement = (
```
Comment (brionmario) [inline packages/react/src/components/ActionCard/__tests__/ActionCard.test.tsx:25]: Better if the component starts with a capital.

File: packages/react/src/components/ActionCard/ActionCard.tsx [hunk 6]
```diff
@@ -0,0 +1,108 @@
...
+          }}
+        >
+          {title}
+        </Typography>
+        <Typography
+          variant="body1"
+          color="secondary.main"
+          sx={{
+            color: '#8D91A3',
+            fontFamily: 'Montserrat',
```
Comment (brionmario) [inline packages/react/src/components/ActionCard/ActionCard.tsx:85]: Glimer with be inherited right?

File: packages/react/src/components/ActionCard/ActionCard.tsx [hunk 7]
```diff
@@ -0,0 +1,108 @@
...
+            fontSize: '15px',
+            fontStyle: 'normal',
+            fontWeight: 400,
+            lineHeight: '24px',
+          }}
+        >
+          {title}
+        </Typography>
+        <Typography
+          variant="body1"
```
Comment (brionmario) [inline packages/react/src/components/ActionCard/ActionCard.tsx:81]: Shall we try with `body2` and see if we can get rid of the custom styles

Comment (thivi) [reply packages/react/src/components/ActionCard/ActionCard.tsx]: It doesn't work as intended.

**Review by brionmario — APPROVED** [review]

**Pull request merged**